compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1158,6 +1158,7 @@ class OMR_EXTENSIBLE CodeGenerator
    bool needRelocationsForStatics() { return false; }
    bool needRelocationsForBodyInfoData() { return false; }
    bool needRelocationsForPersistentInfoData() { return false; }
+   bool helpersNeedRelocation() { return false; }
```

Comment:
This new API is inconsistent with the other related APIs. Do you agree that `needRelocationsForHelpers` may be a better name? Let's also document the new API.